### PR TITLE
Modified README to add  API and added scroll ability to text.

### LINKED ISSUE #120: Text Scrolling
screen shot 2017-07-24 at 5 27 37 pm 1
Currently content in vis.text if larger than window, then window needs to be dragged and resized. 

But instead we can have scrollable (in both x and y) window for text. So whenever content is larger than window, scroll is much better option than resizing windows. 

If this sounds reasonable then I can send a PR for this. 

On sidelines, pip package is not updated with latest additions to this package. Filter and zoom in additions are tremendously helpful. So can you let us know how is the release cycle for visdom.

### PR DESCRIPTION
**Pull request by shubhamjain0594**
- README.md modified for `images` API inclusion
- Fix #120

### CODE REVIEW

File: js/TextPane.js [hunk 1]
```diff
@@ -23,7 +23,9 @@ class TextPane extends React.Component {
   render() {
     return (
       <Pane {...this.props} handleDownload={this.handleDownload}>
-        <div dangerouslySetInnerHTML={{__html: this.props.content}}></div>
+        <div className="text-pane"><div
```
Comment (ajabri) [inline js/TextPane.js]: Can we make the className "content-text"? To match with "content-image" :)

**Comment by ajabri** [comment]
Other than the above comment, looks good :)

**Comment by shubhamjain0594** [comment]
@ajabri Comments have been addressed.

**Comment by ajabri** [comment]
Thanks @shubhamjain0594 !

**Pull request closed without merging**